pkg/config/templates/rancherd-15-logging.yaml

```diff
@@ -0,0 +1,95 @@
+resources:
+- apiVersion: management.cattle.io/v3
+  kind: ManagedChart
+  metadata:
+    name: rancher-logging
+    namespace: fleet-local
+  spec:
+    chart: rancher-logging
+    releaseName: rancher-logging
+    version: {{ .LoggingChartVersion }}
+    defaultNamespace: cattle-monitoring-system
+    repoName: harvester-charts
+    diff:
+      comparePatches:
+        - apiVersion: admissionregistration.k8s.io/v1
+          kind: ValidatingWebhookConfiguration
+          name: rancher-logging-admission
+          operations:
+            - { "op": "remove", "path": "/webhooks" }
+        - apiVersion: admissionregistration.k8s.io/v1
+          kind: MutatingWebhookConfiguration
+          name: rancher-logging-admission
+          operations:
+            - { "op": "remove", "path": "/webhooks" }
+    targets:
+      - clusterName: local
+        clusterSelector:
+          matchExpressions:
+            - key: provisioning.cattle.io/unmanaged-system-agent
+              operator: DoesNotExist
+    values:
+      fluentd:
+        fluentdPvcSpec:
+          storageClassName: longhorn
+      alertmanager:
```

Comment:
is this file copied from `rancherd-13-monitoring`, there should no `alertmanager`, `grafana`

Reply:
removed the unnecessary values from `rancherd-15-logging.yaml`